content/en/_index.md

```diff
@@ -43,24 +50,28 @@ security. The Kyverno CLI can be used to test policies and validate resources as
 
 {{% blocks/lead color="gray" %}}
 
+<br/>
+
 # Join our community { class="text-center" }
 
-[![Go Report Card](https://goreportcard.com/badge/github.com/kyverno/kyverno)](https://goreportcard.com/report/github.com/kyverno/kyverno) 
-[![License: Apache-2.0](https://img.shields.io/github/license/kyverno/kyverno?color=blue)](https://github.com/kyverno/kyverno/)
-[![GitHub Repo stars](https://img.shields.io/github/stars/kyverno/kyverno)](https://github.com/kyverno/kyverno/stargazers)
+<br/>
+
+### Interested in learning and contributing ?
```

Comment:
Headings shouldn't have punctuation.

Reply:
Can you suggest a replacement ?

Reply:
Actually, this is fine. No space before punctuation though.

Reply:
Fixed